.ci/merge-fixes.sh

```diff
@@ -19,7 +19,7 @@ else
         $GH pr checkout -b pr-$a $a
         git fetch --unshallow --all
         git checkout -q test_head
-        if git merge --no-edit --squash -q pr-$a; then
+        if git merge --no-edit --squash --allow-unrelated-histories -q pr-$a; then
```

Comment:
if one has to use this flag, typically something's wrong already

Reply:
The checkout action results in a detached head for PRs. You can call that "wrong" but its by design.